services/frontend-v3/src/components/orgTree/OrgTreeView.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Tree } from "antd";
+import PropTypes from "prop-types";
+
+import { useSelector } from "react-redux";
+
+const OrgTreeView = ({ data }) => {
+  const { locale } = useSelector((state) => state.settings);
+
+  const treeData = [];
+
+  const titleString = (title) => {
+    if (typeof title === "object") {
+      return title[locale];
+    }
+    return title;
+  };
+
+  const genTreeBranch = (orgData) => {
+    let retVal = [];
+    const branchSize = orgData.length;
+    for (let i = 0; i < branchSize; i += 1) {
+      const val = orgData[branchSize - i - 1];
+      retVal = {
+        title: titleString(val.title),
+        key: val.id,
+        children: [retVal],
+      };
```

Comment:
To not see the --- at the bottom of the tree, the children value of the leaf of tree should not be there, maybe doing something like this?

```suggestion
      const object = {
        title: titleString(val.title),
        key: val.id,
      };

      if (retVal.length !== 0) {
        object.children = [retVal];
      }

      retVal = object;
```

Reply:
Changed to conditionally include children